### PR TITLE
Issue 6/percentage home wins && Issue 7 percentage visitor wins

### LINKED ISSUE #6: Create percentage_home_wins method
Percentage of games that a home team has won (rounded to the nearest 100th)

Return Value - Float

### LINKED ISSUE #8: Create percentage_visitor_wins method
Percentage of games that a visitor has won (rounded to the nearest 100th)

Return Value - Float

### PR DESCRIPTION
**Pull request by JoseLopez235**
This PR contains:
- test and method for a percentage of home wins
- test and method for a percentage of visitor wins
- all test passing

closed #6 
closed #8

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -46,10 +46,10 @@ def initialize(locations = nil)
     csv_game_files
     csv_team_files
     csv_game_team_files
-    @highest_total_score_stat  = nil
-    @lowest_total_score        = nil
-    @percentage_home_wins      = nil
-    @percentage_visitor_wins   = nil
+    @highest_total_score_stat     = nil
+    @lowest_total_score_stat      = nil
```
Comment (foymikek) [inline lib/stat_tracker.rb]: Do we need to change the instance variable names? Arn't they meant to be labelled the same?

**Review by foymikek — COMMENTED** [review]
Noticed you had changed the stat_tracker instance variable names, are they meant to be different?

File: lib/stat_tracker.rb (hunk 1 above)
Comment (JoseLopez235) [reply lib/stat_tracker.rb]: no keep it the same for the harness can detect it

**Review by foymikek — APPROVED** [review]
LGTM

**Pull request merged**